gym_sts/spaces/constants.py

```diff
@@ -1009,20 +1012,45 @@
     "FaceTrader",
 ]
 
+ALL_MAP_LOCATIONS = [
+    "NONE",  # Indicates the absence of node
+    "M",  # Monster
+    "?",  # Unknown
+    "$",  # Shop
+    "E",  # Elite
+    "B",  # Burning Elite. Note that this symbol isn't actually used by the game.
+    "T",  # Treasure
+    "R",  # Rest site
+]
+NUM_MAP_LOCATIONS = len(ALL_MAP_LOCATIONS)
+NUM_MAP_NODES_PER_ROW = 7
+NUM_MAP_ROWS = 15
+NUM_MAP_NODES = NUM_MAP_NODES_PER_ROW * NUM_MAP_ROWS
+NUM_MAP_EDGES_PER_NODE = 3  # Max branching factor from one layer to the next
+NUM_MAP_EDGES = NUM_MAP_NODES_PER_ROW * NUM_MAP_EDGES_PER_NODE * (NUM_MAP_ROWS - 1)
```

Comment:
May want to add a brief refresher how edges are stored as multibinary here - seems forgettable